tests/pl/cover_HassCloseCover.yaml

```diff
@@ -27,6 +27,7 @@ tests:
       - "Zasłoń rolety w sypialni"
       - "Zasłoń wszystkie rolety w sypialni"
       - "Zamknij w sypialni wszystkie rolety"
+      - "Przymknij całe rolety w sypialni"
```

Comment:
The grammar here is not correct.
One can say "Zasłoń wszystkie rolety w sypialni" to close all the roller shutters in the sleeping room, but "Przymknij całe rolety w sypialni" is confusing (at least to me).
Personally for me "Przymknij rolety w sypialni" means that I'd like to close (lower) roller shutters to 20% or 10% (almost closed), but I think this use case isn't supported (yet)

Reply:
Yes it was also my feeling but it was decided in one the previous PR to let it through because we focus on flexibility right now. Maybe in different regions of Poland it means something different but at my location using "Przymknij" as "Zamknij" is not common (at least I've never heard it).

This way or another, this was not added in this PR, I'm just adding the tests now ;)

Reply:
Hey, in my region we use sentence "Przymknij wszystkie rolety w sypialni". 😉 The sentence "Przymknij całe rolety w sypialni" is little weird..